lib/text_diff.ml

```diff
@@ -36,6 +43,18 @@ let process_value_diff (val_diff : Diff.value) =
           };
       ]
 
+let process_type_diff (type_diff : Diff.type_) =
+  match type_diff.tdiff with
+  | Added td ->
+      [
+        Diffutils.Diff.Diff { orig = []; new_ = td_to_lines type_diff.tname td };
+      ]
+  | Removed td ->
+      [
+        Diffutils.Diff.Diff { orig = td_to_lines type_diff.tname td; new_ = [] };
+      ]
+  | Modified _ -> failwith "Unimplemented"
```

Comment:
Even though the detection of modified types is not implemented, the printing of modified type is simple enough to be implemented now!